scripts/generate-icon-react-wrapper.js

```diff
@@ -13,6 +13,7 @@ governing permissions and limitations under the License.
 
 import { generateIconWrapper } from './cem-plugin-react-wrapper.js';
 
+console.log(`hola`);
```

Comment:
What does this change have to do with changesets?

Reply:
done!